src/uct/tl/context.h

```diff
@@ -31,13 +32,24 @@ struct uct_pd_component {
 
     const char             name[UCT_PD_COMPONENT_NAME_MAX];
     void                   *priv;
+    const char             *cfg_prefix;        /**< Prefix for configuration environment vars */
+    ucs_config_field_t     *pd_config_table;   /**< Defines PD configuration options */
+    size_t                 pd_config_size;     /**< PD configuration structure size */
     size_t                 rkey_buf_size;
-    ucs_list_link_t        tl_list;  /* List of uct_pd_registered_tl_t */
+    ucs_list_link_t        tl_list;            /* List of uct_pd_registered_tl_t */
     ucs_list_link_t        list;
 };
 
 
 /**
+ * "Base" structure which defines PD configuration options.
+ * Specific PDs extend this structure.
+ */
+struct uct_pd_config {
```

Comment:
need to add some dummy field, because C and C++ treat empty structures diferently